vinyldns/resource_record_set.go

```diff
@@ -204,16 +205,24 @@ func resourceVinylDNSRecordSetDelete(d *schema.ResourceData, meta interface{}) e
 
 	deleted, err := meta.(*vinyldns.Client).RecordSetDelete(d.Get("zone_id").(string), rsID)
 	if err != nil {
-		return err
+		if vErr, ok := err.(*vinyldns.Error); ok {
```

Comment:
this production code update does not look to alter how updates are done, rather adds some nice things around failure on recordset delete

Reply:
@pauljamescleary Sorry - could you rephrase this and elaborate? I'm not sure I follow.

Reply:
Does my PR message clarify?

https://github.com/vinyldns/terraform-provider-vinyldns/pull/37#issue-281561089